docker/Ansible/ansible_container.yaml

```diff
@@ -0,0 +1,122 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://cloudify.co/spec/cloudify/5.0.5/types.yaml
+  - plugin:cloudify-docker-plugin
+
+inputs:
+
+  docker_host:
+    type: string
+    default: ''
+
+  docker_user:
+    type: string
+    default: centos
+
+  ansible_host:
+    type: string
+    default: 10.239.3.26
+
+  ansible_user:
+    type: string
+    default: ubuntu
+
+  playbook_path:
+    type: string
+    default: 'apache2/playbook.yaml'
+
+  container_volume:
+    description: volume that will be mapped to container
+    type: string
+    default: "/ansible-playbooks"
+
+  ansible_version:
+    description: Ansible version to prepare image for
+    type: string
+    default: "2.9.5"
+
+  image_tag:
+    description: Tag for the image to build
+    type: string
+    default: "ansible:"
+
+dsl_definitions:
+
+  docker_config: &docker_config
+    docker_host: { get_input: docker_host }
+    docker_rest_port: 2375
+
+node_templates:
+
+  docker_ansible_image:
+    type: cloudify.nodes.docker.image
+    properties:
+      client_config: *docker_config
+      resource_config:
+        image_content:
+          concat:
+            - |
+              FROM amd64/centos:7
+              MAINTAINER Cosmo (hello@cloudify.co)
+              RUN yum install -y openssh-server openssh-clients
+              RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
+              RUN python get-pip.py
+              RUN pip install --upgrade pip==9.0.1
+            - |-
+              RUN pip install ansible==
+            - { get_input: ansible_version }
+        tag:
+          concat:
+            - { get_input: image_tag }
+            - { get_input: ansible_version }
```

Comment:
How long does this take to build?

Reply:
it depends on docker host internet connection -but in general not long-